dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/VectorSearch/VectorSearchWithFilterConformanceTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Support;
+using VectorDataSpecificationTests.Xunit;
+using Xunit;
+
+namespace VectorDataSpecificationTests.VectorSearch;
+
+public abstract class VectorSearchWithFilterConformanceTests<TKey>(VectorStoreFixture fixture) where TKey : notnull
```

Comment:
nit: this can use VectorStoreCollectionFixture. But in any case, I'll be doing a cleanup of the tests after Build - this definitely looks good enough for now.